sdk/dotnet/Pulumi.Automation/UpdateOptions.cs

```diff
@@ -15,5 +16,9 @@ public class UpdateOptions
         public string? Message { get; set; }
 
         public List<string>? Target { get; set; }
+
+        public Action<string>? OnOutput { get; set; }
+
+        public Action<string>? OnStdErr { get; set; }
```

Comment:
For consistency, we should go with `OnOutput` and `OnError` or OnStandardOutput` and `OnStandardError`. Also, we should not use abbreviations in .NET.

Reply:
I think `OnError` should be sufficient, maybe with an added doc comment 

```
/// <summary>
/// Optional callback which is invoked whenever StandardError is written into
/// </summary>
public Action<string>? OnError { get; set; }
```

Reply:
Hey! I would explicitly like to avoid calling this `OnError` because it is a misnomer for the actual functionality. We are piping Standard Error output here, `onError` implies that this will only be triggered on actual errors, but conventionally standard error is not only used for errors and Pulumi also logs debug output to standard error. 

We should have originally named `OnOutput` to be `OnStandardOutput`, that was a miss on our part. I would, however, like to avoid continuing down the misnaming path just because we have done so previously.

Reply:
Fair enough, I'll rename both of these